### PR TITLE
[CUBRIDMAN-19] update jdbc_driver usage

### PR DESCRIPTION
**Pull request by kmjc6611**
http://jira.cubrid.org/browse/CUBRIDMAN-19

### CODE REVIEW

**Comment by kmjc6611** [comment]
I excluded changes about python.rst to make a new PR for python.rst.

**Review by mhoh3963 — DISMISSED** [review]

**Review by zionyun — DISMISSED** [review]

**Review by charischau — DISMISSED** [review]

File: en/api/jdbc.rst [hunk 1]
```diff
@@ -664,15 +664,15 @@ The line "a" in the example 1 is where data of collection types (**SET**, **MULT
...
-                   oid.removeFromSequence("listtest",1);
+				   oid.addToSet("settest", Integer.valueOf(10));
+				   oid.addToSet("multisettest", Integer.valueOf(20));
+				   oid.addToSequence("listtest", 1, Integer.valueOf(30));
+				   oid.addToSequence("listtest", 100, Integer.valueOf(100));
+				   oid.putIntoSequence("listtest", 99, Integer.valueOf(99));
+				   oid.removeFromSet("settest", Integer.valueOf(1));
+				   oid.removeFromSet("multisettest", Integer.valueOf(2));
+				   oid.removeFromSequence("listtest", 99);
+				   oid.removeFromSequence("listtest", 1);
```
Comment (hgryoo) [inline en/api/jdbc.rst]: please fix the broken indentation in this example. you need to build the manual and check on a web browser after you revise.

File: ko/api/jdbc.rst [hunk 2]
```diff
@@ -667,15 +667,15 @@ OID를 사용할 때 다음의 규칙을 지켜야 한다.
...
-                   oid.removeFromSequence("listtest",1);
+				   oid.addToSet("settest", Integer.valueOf(10));
+				   oid.addToSet("multisettest", Integer.valueOf(20));
+				   oid.addToSequence("listtest", 1, Integer.valueOf(30));
+				   oid.addToSequence("listtest", 100, Integer.valueOf(100));
+				   oid.putIntoSequence("listtest", 99, Integer.valueOf(99));
+				   oid.removeFromSet("settest", Integer.valueOf(1));
+				   oid.removeFromSet("multisettest", Integer.valueOf(2));
+				   oid.removeFromSequence("listtest", 99);
+				   oid.removeFromSequence("listtest", 1);
```
Comment (hgryoo) [inline ko/api/jdbc.rst]: please fix the broken indentation in this example. you need to build the manual and check on a web browser after you revise.

**Review by hgryoo — CHANGES_REQUESTED** [review]

File: en/api/jdbc.rst (hunk 1 above)
Comment (kmjc6611) [reply en/api/jdbc.rst]: I check it and commit after build the manual.

**Review by nays111 — APPROVED** [review]

File: ko/api/jdbc.rst (hunk 2 above)
Comment (kmjc6611) [reply ko/api/jdbc.rst]: I check it and commit after build the manual.

**Review by kmjc6611 — COMMENTED** [review]
I check it

**Review by hgryoo — APPROVED** [review]

**Comment by kmjc6611** [comment]
http://jira.cubrid.org/browse/CUBRIDMAN-19

Fixed wrong  contents in ko,en/jdbc.rst
Changed to latest version of CUBRID
Since Integrer is no longer used, i changed it to using valueOf

**Pull request merged**